Assets/MRTK/SDK/Features/UX/Scripts/Pointers/TeleportPointer.cs

```diff
@@ -118,10 +118,33 @@ private float InputThresholdSquared
             }
         }
 
+
+        #region Audio Management
+        [Header("Audio management")]
+        [SerializeField]
+        private AudioSource PointerAudioSource = null;
```

Comment:
nit on [the capitalization](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#naming-conventions) of these private fields (camelCase vs PascalCase)